README.md

```diff
@@ -29,6 +29,7 @@ A curated list of awesome things related to [Chart.js](https://www.chartjs.org)
 - [crosshair](https://github.com/abelheinsbroek/chartjs-plugin-crosshair) - Adds a data crosshair to line and scatter charts.
 - [datalabels](https://github.com/chartjs/chartjs-plugin-datalabels) - Displays labels on data for any type of charts.
 - [deferred](https://github.com/chartjs/chartjs-plugin-deferred) -  Defers initial chart update until chart scrolls into viewport.
+- [dragData](https://github.com/chrispahm/chartjs-plugin-dragData) - Lets users drag data points on the chart.
```

Comment:
The plugin name and package name [should be lowercase](https://www.chartjs.org/docs/latest/developers/plugins.html#plugin-id): 

```
[dragdata](https://github.com/chrispahm/chartjs-plugin-dragdata)
```

Can you also add an id to [your plugin](/chrispahm/chartjs-plugin-dragData/blob/master/src/index.js#L157) that reflects the package name: `id: 'dragdata'` (similar to [this](/chartjs/chartjs-plugin-datalabels/blob/master/src/plugin.js#L123))

Reply:
Thanks @simonbrunel for having a look!
I updated the plugin's name to be all lowercase, and added the required plugin id (see https://github.com/chrispahm/chartjs-plugin-dragdata/pull/28).
Also, the plugin name in this PR was updated accordingly. I hope this addresses all the issues mentioned?